docker-compose.yml

```diff
@@ -10,8 +10,9 @@ services:
     restart: always
     # To run from this source code instead of Docker hub,
     # disable "image:"" and enable "build:" below
-    image: commonworkflowlanguage/cwlviewer:v1.3.0
-    #build: .
+    #image: commonworkflowlanguage/cwlviewer:v1.3.0
+    image: commonworkflowlanguage/cwlviewer
+    build: .
```

Comment:
Isn't this what `docker-compose.override.yml` is for? https://github.com/common-workflow-language/cwlviewer#recommended---running-with-docker

Reply:
Yes but right the minute I'm just trying to get anything to work and I don't understand why it isn't updating